eole/transforms/tokenize_id.py

```diff
@@ -92,6 +92,9 @@ def tokenize_string(self, string, side="src", is_train=False):
             kwargs = {"max_length": self.max_length, "truncation": True}
         else:
             kwargs = {}
+        string = string.replace(DefaultTokens.SEP, "\n").replace(
+            DefaultTokens.MASK_BEFORE, self.tokenizers[side].pad_token
+        )
```

Comment:
shouldn't we handle it the same way for other tokenizers ?

Reply:
I have only tested for that one (with eurollm) at this point; an error will be raised with the others.

Reply:
In fact the logic to apply id_tokenization already exists here
https://github.com/eole-nlp/eole/blob/40ed6ea8c967107ac7da33aa926433845ee2d268/eole/predict/inference.py#L196-L200
The point is that the "filtertoolong" transform shouldn't be applied after "huggingface_tokenize". See https://github.com/eole-nlp/eole/issues/191
I remove this.